.github/workflows/ci.yml

```diff
@@ -1,4 +1,4 @@
-on: [push]
+on: [pull_request]
```

Comment:
> Based on GitHub docs, it seems that activity types opened, synchronize, or reopened trigger this action. I prefer to specify them explicitly because of understandability. Could you tell me what you think about it?

Either is fine for me. Will do.

> By the way, I feel this change does not align PR title. Changing PR title or split PR is a good option.

Hmm, that makes sense. I'd prefer to make the CI keep running on this PR and this PR has already included some tiny commits that may not worth splitting across PRs. So let me just rename the PR title here.